automation/project/Dependencies.scala

```diff
@@ -55,7 +55,7 @@ object Dependencies {
     "org.scalatest"       %%  "scalatest"     % "3.2.2"   % "test",
     "org.seleniumhq.selenium" % "selenium-java" % "3.8.1" % "test",
     "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
-    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-bef373fbd-SNAP"
+    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-384ab501b"
```

Comment:
should this be <hash>-SNAP?

Reply:
Nope, `-SNAP` indicates that it's a snapshot which we actually prefer not to use. This was a snapshot before in order to simplify the PPW merge process, but ideally we aren't using snapshots in our main branches

Reply:
The `-SNAP` above (for `serviceTestV` will also be removed once the workbench-libs PR has been merged)